pkg/config/config_template.yaml

```diff
@@ -45,6 +45,21 @@ api_key:
 #   - env:prod
 #   - role:database
 
+# Split tags according to given string.
+# Supported only for tags handled by the Tagger (Kubernetes, Docker, AWS ECS, ...)
```

Comment:
Rewording suggestions: `For now, only container tags support this feature`, same for the changelog

Reply:
Thanks for the suggestion, but I'd like this PR to also include node label processing, as you pointed above. As a result, `container tags` may not be the proper wording here?